tests/integration/test_parity.py

```diff
@@ -149,20 +149,26 @@ def passwordfile():
 def parity_process(
         parity_import_blocks_process,
         parity_binary,
-        ipc_path, datadir,
+        ipc_path,
+        datadir,
         passwordfile,
         author):
-
-    run_parity_command = (
+    command_arguments = (
         parity_binary,
         '--chain', os.path.join(datadir, 'chain_config.json'),
         '--ipc-path', ipc_path,
-        '--base-path', str(datadir),
-        '--unlock', str(author),
-        '--password', str(passwordfile),
+        '--base-path', datadir,
+        '--unlock', author,
+        '--password', passwordfile,
     )
+    time.sleep(1)
```

Comment:
Isn't there a deterministic way to do this check? I'd hate to have tests randomly fail if it happens to take too long.

Maybe we can test for the presence of a file every n ms or something?